packages/calcite-components/src/components/shell/shell.tsx

```diff
@@ -193,6 +193,13 @@ export class Shell implements ConditionalSlotComponent {
 
     const contentContainerKey = "content-container";
 
+    const shellPanels: Array<HTMLCalciteShellPanelElement> = Array.from(
```

Comment:
fixme: This would't work in cases when shell-panels are slotted in after the shell has been initialized. Instead, an `onSlotChange` should be used to find shell-panels.